lib/course.rb

```diff
@@ -1 +1,23 @@
+class Course
+  attr_reader :name,
+              :capacity,
+              :students
 
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+  end
+
+  def enroll(student)
+    @students << student
+  end
+
+  def full?
+    if @students.count >= @capacity
+      true
+    else
+      false
+    end
+  end
```

Comment:
This works. You could simplify the syntax further:
```suggestion
  def full?
    @students.count >= @capacity
  end
```